stdlib/src/utils/string_slice.mojo

```diff
@@ -301,14 +262,12 @@ struct StringSlice[is_mutable: Bool, //, origin: Origin[is_mutable].type,](
         #   StringLiteral is guaranteed to use UTF-8 encoding.
         # FIXME(MSTDL-160):
         #   Ensure StringLiteral _actually_ always uses UTF-8 encoding.
-        # TODO(#933): use when llvm intrinsics can be used at compile time
+        # FIXME: this gets practically stuck at compile time
```

Comment:
**Suggestion** Can you please file a new GitHub issue for the compiler team to look into and extend @lattner's work for running the llvm intrinsics at comp time?  Thanks!

Reply:
opened issue #3706